src/pages/workspace/accounting/intacct/IntacctPrerequisitesPage.tsx

```diff
@@ -0,0 +1,105 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React, {useMemo, useRef} from 'react';
+import {View} from 'react-native';
+// eslint-disable-next-line no-restricted-imports
+import type {GestureResponderEvent, Text as RNText} from 'react-native';
+import Computer from '@assets/images/computer.svg';
+import Button from '@components/Button';
+import FixedFooter from '@components/FixedFooter';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import * as Expensicons from '@components/Icon/Expensicons';
+import ImageSVG from '@components/ImageSVG';
+import MenuItemList from '@components/MenuItemList';
+import ScreenWrapper from '@components/ScreenWrapper';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import type {SettingsNavigatorParamList} from '@libs/Navigation/types';
+import * as ReportActionContextMenu from '@pages/home/report/ContextMenu/ReportActionContextMenu';
+import * as Link from '@userActions/Link';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
+
+type IntacctPrerequisitesPageProps = StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.ACCOUNTING.SAGE_INTACCT_PREREQUISITES>;
+
+function IntacctPrerequisitesPage({route}: IntacctPrerequisitesPageProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const popoverAnchor = useRef<View | RNText | null>(null);
+    const policyID = route.params.policyID;
+
+    const menuItems = useMemo(
+        () => [
+            {
+                title: translate('workspace.intacct.downloadExpensifyPackage'),
+                key: 'workspace.intacct.downloadExpensifyPackage',
+                icon: Expensicons.Download,
+                iconRight: Expensicons.NewWindow,
+                shouldShowRightIcon: true,
+                action: () => {
+                    Link.openExternalLink(CONST.EXPENSIFY_PACKAGE_FOR_SAGE_INTACCT);
+                    return Promise.resolve();
+                },
```

Comment:
Check out this video, I don't think it's working now:

https://github.com/Expensify/App/assets/16502320/115ee789-6632-48a4-93e2-2ac92c3e820b